inc/saiport.h

```diff
@@ -89,6 +106,8 @@ typedef struct _sai_port_oper_status_notification_t
     /** Port operational status */
     sai_port_oper_status_t port_state;
 
+    /** Bitmap of various port error or fault status */
+    sai_port_error_status_t port_error_status;
```

Comment:
There could be multiple error status' active at a time. For example, a FEC sync loss should also show fire a local fault. How would we capture that here?

Reply:
it is defined as bitmap, so multiple bits can be set